### PR TITLE
Slack example

### PR DESCRIPTION
**Pull request by lagache**

### CODE REVIEW

File: examples/message/SlackMessageExample.js [hunk 1]
```diff
@@ -0,0 +1,94 @@
+const { Reshuffle } = require('reshuffle')
+const { SlackConnector, SlackEventType, SlackEvents } = require('reshuffle-slack-connector')
+const app = new Reshuffle()
+
+const main = async () => {
+  const slackConnector = new SlackConnector(app, {
```
Comment (ashevat) [inline examples/message/SlackMessageExample.js:6]: can you add a link to where to get these tokens from?

File: examples/message/SlackMessageExample.js [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+    token: '<your_slack_token_starting_with_xox>',
+    signingSecret: '<your_slack_signing_secret>',
+    port: '<slack_receiver_port>', // Default to 3000
+    endpoints: '<slack_receiver_endpoints>', // Default to '/'
+  })
+
+  // Events
```
Comment (ashevat) [inline examples/message/SlackMessageExample.js:13]: probably needs documentation on how to set these events up in Slack

File: examples/message/SlackMessageExample.js [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+  slackConnector.on(
+    {
+      type: SlackEventType.EVENT,
```
Comment (ashevat) [inline examples/message/SlackMessageExample.js:16]: SlackEventType.EVENT sounds confusing

File: examples/message/SlackMessageExample.js (hunk 1 above)
Comment (lagache) [reply examples/message/SlackMessageExample.js:6]: added

File: examples/message/SlackMessageExample.js (hunk 2 above)
Comment (lagache) [reply examples/message/SlackMessageExample.js]: added

File: examples/message/SlackMessageExample.js (hunk 3 above)
Comment (lagache) [reply examples/message/SlackMessageExample.js:26]: I kept the same semantic as runtime v1. I don't have a better name to suggest but I think it makes sense when you look at the other types

**Review by ashevat — APPROVED** [review]

**Pull request merged**